BridgeAppSDK/DataArchiving/SBAEncryptionHelper.swift

```diff
@@ -40,5 +40,36 @@ import UIKit
         let certificatePath = NSBundle.mainBundle().pathForResource(sharedAppDelegate.bridgeInfo.certificateName, ofType: "pem")
         return certificatePath
     }
+    
+    class func encryptedDataFilename() -> String {
+        return "encrypted.zip"
+    }
+    
+    class func encryptedDataPathRoot() -> String {
+        return NSTemporaryDirectory()
+    }
+    
+    public class func encryptedFilesAwaitingUploadResponse() -> [String] {
+        let tmpDir = encryptedDataPathRoot()
+        let fileMan = NSFileManager.defaultManager()
+        
+        let tmpContents = fileMan.subpathsAtPath(tmpDir)
+        let filesOfInterest = tmpContents?.mapAndFilter({ (file) -> String? in
+            if (file as NSString).lastPathComponent != encryptedDataFilename() { return nil }
```

Comment:
`guard (file as NSString).lastPathComponent == encryptedDataFilename() else { return nil }`

I've been trying to use the `guard` for early exits since this seems to be the Apple-preferred pattern.